tests/core/test_support_errno.c

```diff
@@ -9,10 +9,10 @@
 #include <errno.h>
 #include <string.h>
 #include <sys/types.h>
-#include <unistd.h>
+#include <time.h>
 
 int main() {
-  pid_t rtn =  fork();
```

Comment:
The SUPPORT_ERRNO flag only effects that JS code so this test must call an errno-setting JS functions.   Since this PR moved fork() from JS to native code we can't use `fork` here anymore.